app/assets/stylesheets/styles/landing.scss

```diff
@@ -0,0 +1,176 @@
+.landing {
+  $ts-p-offset: 1px;
+  $ts-n-offset: -1px;
+  $ts-blur: 7px;
+  $ts-color: rgba(0,0,0,0.4);
+  text-shadow: $ts-n-offset $ts-n-offset $ts-blur $ts-color,
+                $ts-n-offset $ts-p-offset $ts-blur $ts-color,
+                $ts-p-offset $ts-n-offset $ts-blur $ts-color,
+                $ts-p-offset $ts-p-offset $ts-blur $ts-color;
+}
+
+.landing__sign-in {
+  position: absolute;
+  top: 0;
+  right: 0;
+  padding: 0.7em;
+
+  @media #{$medium-up} {
+    text-shadow: none;
+    padding: 1em;
+
+    a {
+      @include button($padding: $button-tny, $bg: $secondary-color);
+
+      @media #{$xlarge-up} {
+        font-size: 1rem;
+      }
+    }
+  }
+
+}
+
+.landing__hero {
+  padding: 2.3em 0.5em 8%;
+
+  @media #{$large-up} {
+    padding: 2.3em 5% 5%;
+  }
+}
+
+.hero {
+  @media #{$medium-up} {
+    margin: 7% auto;
+    max-width: 24em;
+  }
+
+  @media #{$large-up} {
+    font-size: 3rem;
+  }
+
+  @media #{$xlarge-up} {
+    font-size: 4rem;
+  }
+}
+
+.hero__first {
+  display: block;
+  margin-bottom: 0.5em;
+  text-align: left;
+}
+
+.hero__second {
+  display: block;
+  text-align: right;
+}
+
+.landing__description {
+  background-color: rgba(0,0,0,0.7);
+  color: $base2;
+  padding: 0.5em 7%;
+  text-align: center;
+  $ts-p-offset: 1px;
+  $ts-n-offset: -1px;
+  $ts-blur: 3px;
+  $ts-color: rgba(0,0,0,0.2);
+  text-shadow: $ts-n-offset $ts-n-offset $ts-blur $ts-color,
+                $ts-n-offset $ts-p-offset $ts-blur $ts-color,
+                $ts-p-offset $ts-n-offset $ts-blur $ts-color,
+                $ts-p-offset $ts-p-offset $ts-blur $ts-color;
+
+  @media #{$medium-up} {
+    font-size: 1.1em;
+    padding: 1em 5%;
+
+    p {
+      display: inline-block;
+      font-size: 1em;
+      margin-bottom: 0;
+      width: 46.8%; //2*47 + 5.55 = 100%
```

Comment:
Properties should be ordered display, font-size, margin-bottom, vertical-align, width